ch04.adoc

```diff
@@ -216,46 +216,54 @@ double time(time) ;
+We recommend that **`year`** and **`month`** should not be used, because of the potential for mistakes and confusion.
 
 UDUNITS defines a **`minute`** as 60 **`seconds`**, an **`hour`** as 3600 **`seconds`** and a **`day`** as 86400 **`seconds`**.
 These are not calendar units.
-When a leap second is inserted into UTC, the minute, hour and day affected differ by one second from their usual durations according to clock time, but the UDUNITS and CF **`minute`**, **`hour`** and **`day`** do not; they are fixed units of measure.
-See also <<leap-seconds>>.
+When a leap second is inserted into UTC, the minute, hour and day affected differ by one second from their usual durations according to clock time, but the units of **`minute`**, **`hour`** and **`day`** do not; they are fixed units of measure.
+To avoid mistakes and confusion, we therefore recommend that these units should not be used in the **`utc`** calendar; instead, we recommend the **`second`** as the unit for time coordinates in that calendar.
+See also <<calendar>> and <<leap-seconds>>.
 
 UDUNITS permits a number of alternatives to the word **`since`** in the units of time coordinates.
 All the alternatives have exactly the same meaning in UDUNITS.
 For compatibility with other software, CF strongly recommends that `since` should be used.
 
 The reference datetime string (appearing after the identifier **`since`**) is required.
-It must include the date, which may optionally be followed by time or time zone offset or both. Its format is __y__-__m__-__d__ [__H__:__M__:__S__] [__Z__], where [...] indicates an optional element:
+It must include the date, which may be followed by time or time zone offset or both.
+Its format is __y__-__m__-__d__ [__H__:__M__:__S__] [__Z__], where [...] indicates an optional element, as follows:
 
 * _y_ is year, _m_ month, _d_ day, _H_ hour and _M_ minute, which are all integers of one or more digits, and _y_ may be prefixed with a sign (but note that some CF calendars do not permit negative years; see <<calendar>>),
 
 * _S_ is second, which may be integer or floating point (see <<leap-seconds>> regarding __S__>59),
 
 * _Z_ is the time zone offset.
-This is an interval of time, specified in one of the formats described below.
-Only numbers (digits, `+`, `-` and `:`) or the letter "Z" are allowed in _Z_, not time zone names or acronyms.
-
+_Z_ is _not_ a time zone name or acronym; it is an interval of time, specified in one of the formats described below.
 The default time zone offset is zero.
+We recommend that a non-zero time zone offset should not be specified, because it is easy to make mistakes about the sign of the offset and allowance for daylight-saving/summer time.
+A non-zero offset is _not allowed_ in the **`utc`** and **`tai`** calendars (see <<calendar>>).
+While the default is an offset of zero, we suggest that a zero offset be stated explicitly to avoid any confusion where omitting it might be misunderstood as indicating local time.
```

Comment:
Just to say that I originally thought "suggest" should be replaced by a BCP14-aligned word, but then I realised that we can't check this, so "suggest" is just fine, I suggest!